sentry-types/src/protocol/v7.rs

```diff
@@ -1648,3 +1648,82 @@ impl<'a> fmt::Display for Transaction<'a> {
         )
     }
 }
+
```

Comment:
please move this into its own file, `v7.rs` is already big enough :-D

Reply:
Heh no problem I did think it was a pretty big boi. :wink: